src/components/C2Table.tsx

```diff
@@ -0,0 +1,143 @@
+import { ChevronDownIcon, ChevronUpIcon } from "@chakra-ui/icons";
+import {
+    FormControl,
+    Table,
+    Thead,
+    Tbody,
+    Tr,
+    Th,
+    Td,
+    Flex,
+    Icon,
+    Input,
+    InputGroup,
+    InputLeftElement,
+} from "@chakra-ui/react";
+import { useMemo, useState } from "react";
+import { FaFilter } from "react-icons/fa";
+import { useTable, useSortBy } from "react-table";
+
+const C2Table = (props) => {
+    const { columns, entries } = props;
+
+    const [data, setData] = useState(entries);
+
+    const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+        useTable(
+            {
+                columns,
+                data,
+            },
+            useSortBy
+        );
+
+    const onFilter = (column, val) => {
+        // if e[column]
+        val = val.toLowerCase();
+
+        if (typeof data[0][column] !== "object") {
+            setData(
+                entries.filter((e) => e[column].toLowerCase().includes(val))
+            );
+            return;
+        }
+
+        // console.log(column, val);
+    };
+
+    const filters = getFilters(columns, onFilter);
```

Comment:
What do the disabled filters do? Do we need them?

Reply:
They are disabled since I can't find a good way to filter input fields. I can make them hidden too but it messes with the table proportions. Another way might be to make them look more disabled.

Reply:
I thin something like this would be better 
<img width="468" alt="image" src="https://user-images.githubusercontent.com/38937780/173200697-be2caaee-7775-45dd-9695-605f17cc3a64.png">

or

<img width="458" alt="image" src="https://user-images.githubusercontent.com/38937780/173200719-52534436-7280-4529-acc3-0a02ec9c3481.png">

Reply:
Hid the filters but left them under headers for now.

Reply:
fair enough